NetworkingSampleApp/NetworkingSampleApp/Scenes/Upload/UploadsView.swift

```diff
@@ -48,7 +30,112 @@ struct UploadsView: View {
                     }
                 }
             }
+
+            multipartUpload
+
+            if !formViewModel.uploadItemViewModels.isEmpty {
+                Section("Multi part upload progress") {
+                    VStack {
+                        ForEach(formViewModel.uploadItemViewModels.indices, id: \.self) { index in
+                            let viewModel = formViewModel.uploadItemViewModels[index]
+                            UploadItemView(viewModel: viewModel)
+                        }
+                    }
+                }
+            }
         }
+        .alert(
```

Comment:
for me it's a bit weirdo having two view models here, but I believe for sample app it's ok

Reply:
Agreed. I could've given more love to the UI part of the example. 😞